src/runner.ts

```diff
@@ -1,28 +1,37 @@
 import axios from './clients/axios';
 import blocks from './clients/blocks';
 import catalog from './clients/catalog';
-import ethereum from './clients/ethereum';
+import evm, { EVMClient } from './clients/evm';
 import ipfs from './clients/ipfs';
 import noizd from './clients/noizd';
 import solana from './clients/solana';
 import sound from './clients/sound';
 import { DBClient, Table } from './db/db';
 import db from './db/sql-db';
+import { Chain, ChainId, ChainType } from './types/chain';
 import { Clients, Processor } from './types/processor';
 
 export const initClients = async (existingDBClient?: DBClient) => {
```

Comment:
is existingDBClient used?

Reply:
yes, it's used here https://github.com/spinamp/spindexer/blob/7b19f9c634263bf994a5754976b5bfc8bd76f878/src/runner.ts#L45